### PR TITLE
Main menu and game instructions

### PR DESCRIPTION
**Pull request by shaw-rohit**
Ready for review!

A main menu has been added where the player is presented with three options:
1. Play game
2. Read instructions
3. Exit

Gameplay instructions have also been added (#1)- quite basic but corresponds to the current game that we have. I will make it more graphical and elaborate once we have an actual hangman figure in the game.

Hangman ASCII art has been added which is visible on the main menu and the instructions page.

`os.system("cls")` has been added at several places and the screen is cleared after every input, mimicking a real game.

`words.py` in the root directory has been deleted. (#21)

### CODE REVIEW

**Comment by arumoy-shome** [comment]
@shaw-rohit Wondering if Menu needs to be it's own class? Shouldn't it just be under Interface?

**Comment by shaw-rohit** [comment]
> @shaw-rohit Wondering if Menu needs to be it's own class? Shouldn't it just be under Interface?

I couldn't make up my mind so I went with a separate class, but we can surely have it under the Interface.

**Comment by arumoy-shome** [comment]
Once #15 merges you probably need to rebase off of master to remove the duplicate code from my branch! 😄 Also, I think we do not need a new class, this should go into interface.

File: hangman/menu.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+class Menu:
+    def __init__(self):
+        pass
+
+    def write_title(self):
```
Comment (arumoy-shome) [inline hangman/menu.py:5]: Perhaps this should replace `interface.write_credit()`?

**Comment by Pungent-Paws** [comment]
Nice, yes, now `''Sorry! -letter- is not in the word! Try again!'' `does show (in cmd). The odd character mentioned in my earlier comment still shows in my IDE but probably that's a non-issue if we instruct testers/users to run the game outside an IDE.

You got my approval to merg this into master :)

**Comment by shrutirao94** [comment]
Hey @shaw-rohit love the interface and ASCII for Hangman! Great work! I would point out two things:

1. `os.system("cls")` is only applicable for windows and does not work on OS X (and perhaps even Linux?) So you might want to incorporate a way to clear screen that works across all platforms! At this point I do see an error for that. (Issue #23 )
2. The game itself is running significantly slower for me. I think this might be because of the clear screen incompatibility!

Other than that, its a +1 from me!

**Comment by shaw-rohit** [comment]
> Hey @shaw-rohit love the interface and ASCII for Hangman! Great work! I would point out two things:
> 
> 1. `os.system("cls")` is only applicable for windows and does not work on OS X (and perhaps even Linux?) So you might want to incorporate a way to clear screen that works across all platforms! At this point I do see an error for that. (Issue #23 )
> 2. The game itself is running significantly slower for me. I think this might be because of the clear screen incompatibility!
> 
> Other than that, its a +1 from me!

The game probably feels slower because of the sleep time that I've added to display the in/correct guess message. Perhaps there's a better way to do this!

**Comment by shaw-rohit** [comment]
Merging this for now. The sleep timer issue can be picked up later!

**Pull request merged**